libs/client-sdk/src/data_sources/polling_data_source.cpp

```diff
@@ -88,6 +88,11 @@ PollingDataSource::PollingDataSource(
     auto const& polling_config = std::get<
         config::shared::built::PollingConfig<config::shared::ClientSDK>>(
         data_source_config.method);
+    if (http_properties.Tls().CustomCAFile()) {
```

Comment:
It's not ideal to have these debug level logs scattered in the polling/streaming/event data sources, I think. 

They will need to be kept in sync, and also would be redundant if we had one for the events system.

Instead, this should be done at a higher level, perhaps the top-level config builder. But there's no logger dependency in there, so it'd need a refactoring.